composer/loggers/in_memory_logger.py

```diff
@@ -83,8 +83,10 @@ def log_table(self,
             raise MissingConditionalImportError(extra_deps_group='pandas',
                                                 conda_package='pandas',
                                                 conda_channel='conda-forge') from e
-        table = pd.DataFrame.from_records(data=rows, columns=columns).to_json(orient='split', index=False)
-        assert isinstance(table, str)
+        table = pd.DataFrame.from_records(data=rows, columns=columns).to_json(orient='split',
+                                                                              index=False,
+                                                                              force_ascii=False)
```

Comment:
why is force ascii false now needed?

Reply:
@bmosaicml added this in his original PR.